src/plugins/gr-he-common/core/shell.py

```diff
@@ -103,5 +103,32 @@ def _setup(self):
                 if not self.environment[ohostedcons.CoreEnv.TMUX_PROCEED]:
                     raise context.Abort('Aborted by user')
 
+        if (
+            not self.environment[ohostedcons.NetworkEnv.FORCE_IPV4]
+            and not self.environment[ohostedcons.NetworkEnv.FORCE_IPV6]
+        ):
+            self.logger.warning(
+                _('It has been detected that either --4 or --6 '
+                  'options were not used.\n '
+                  'On a dual stack environment, '
+                  'it is highly recommended to abort the installation '
```

Comment:
I would just say what's going to happen and let users decide, not recommend anything. "--6" with DNS that keeps returning IPv4 is not going to work anyway.

Just say that on dual stack we're choosing IPv6 but due to our limitation we'll deploy HE as IPv6-only. Make sure that DNS returns only IPv6 or you change java and imageio address resolution. And link to a page (that we don't have now, IIRC) that tells you to change options in config files. 
We are running OST like that successfully, so we can let users do that too,...